### ISSUE TITLE
Null compare bool property changes event is not raised.

### ISSUE THREAD

**Issue by cdytoby** [Fody/PropertyChanged#344]
```
		public DataModel targetData { get; protected set; };
		public bool isDataEmpty
		{
			get
			{
				Console.WriteLine("Feature Test isDataEmpty called.");
				return targetData == null;
			}
		};
		public string features => GetFormattedFeatureString();
		private string GetFormattedFeatureString()
		{
			Console.WriteLine("Feature Test GetFormattedFeatureString called.");
			return targetData?.SomeString;
		}
```
This is a part of my view model, implemented INotifyPropertyChanged.

In this case, after targetData is changed, isDataEmpty is not called, but features is called. Is this a bug, or there is something wrong with this code?

One more thing, how can I check the generated code that gets compiled?

**Comment by SimonCropp** [comment]
u can use ILSpy to look at the resultant code

**Comment by cdytoby** [comment]
Sorry for the trouble, I'm going to close the ticket because I find it hard to understand. After checking the ILSpy, I found out the property does get raised. Then I looked for another half hour and found out: it's the binding issue. The property is correctly binded but if I changed the binded property's target value, in this case, it's a wpf control's visibility, the binding will be broken. I manually changed the visibility and the bind will stop working forever. 

Sorry for the trouble and thanks for the name of the tool @SimonCropp
